packages/melos/lib/src/commands/bootstrap.dart

```diff
@@ -411,3 +420,34 @@ class BootstrapException implements MelosException {
         '${package.path}.';
   }
 }
+
+sealed class BootstrapMode {
+  const BootstrapMode._();
+
+  const factory BootstrapMode.get() = _BootstrapModeGet;
+  const factory BootstrapMode.upgrade() = _BootstrapModeUpgrade;
+  const factory BootstrapMode.downgrade() = _BootstrapModeDowngrade;
+
+  abstract final String subcommand;
+}
+
+final class _BootstrapModeGet extends BootstrapMode {
```

Comment:
I would use an enum for this since it isn't using anything that needs a sealed class.

Reply:
Oh, 🤦‍♂️ 

I forgot those were things.